src/expression.js

```diff
@@ -47,6 +47,22 @@ var getKeyComputeData = function (key, scope, readOptions) {
 		}
 		return res;
 	},
+	// Looks up a value in the result of a Lookup or Call expression
+	lookupValueInLookupOrCall = function(key, lookupOrCall, scope, helperOptions, readOptions) {
+		return {
+			value: compute(function() {
+				var obj;
+
+				if (lookupOrCall instanceof Lookup) {
+					obj = lookupValue(lookupOrCall.key, scope, {}, {}).value();
```

Comment:
this should also be moved out to the outer scope as it's creating  a compute just to run it.